src/rgw/rgw_d3n_cacherequest.h

```diff
@@ -25,51 +24,39 @@ struct D3nGetObjData {
   std::mutex d3n_lock;
 };
 
-class D3nCacheRequest {
-  public:
-    virtual ~D3nCacheRequest() {};
-    virtual void d3n_libaio_release()=0;
-    virtual void d3n_libaio_cancel_io()=0;
-    virtual void d3n_libaio_finish()=0;
-    virtual int d3n_libaio_status()=0;
-};
-
-struct D3nL1CacheRequest : public D3nCacheRequest {
+struct D3nL1CacheRequest {
```

Comment:
it seems like inheritance should have been preserved here, as a foundation for restoring L2?

Reply:
removed it for the sake of simplifying this PR and will re-add it back in the L2 PR